pkg/server/serverpb/status.go

```diff
@@ -87,6 +87,7 @@ type TenantStatusServer interface {
 	// SpanStats is used to access MVCC stats from KV
 	SpanStats(context.Context, *roachpb.SpanStatsRequest) (*roachpb.SpanStatsResponse, error)
 	Nodes(context.Context, *NodesRequest) (*NodesResponse, error)
+	DownloadSpan(ctx context.Context, request *DownloadSpanRequest) (*DownloadSpanResponse, error)
```

Comment:
I mildly want to leave a TODO here.  I know it is a small thing that would require work to change, but in the long run for the final implementation, I don't think we want a method that _changes_ something to be on something called "StatusServer".